site/src/sphinx/server-grpc.rst

```diff
@@ -243,6 +243,26 @@ to all your clients before enabling this flag!
 
 See more details at :ref:`client-grpc`.
 
+Server Reflection
+=================
+
+Armeria supports gRPC server reflection - just add an instance of ``ProtoReflectionService`` to your server.
+
+.. code-block:: java
+
+    import io.grpc.protobuf.services.ProtoReflectionService;
+
+    ServerBuilder sb = new ServerBuilder();
+    ...
+    sb.service(new GrpcServiceBuilder().addService(new MyHelloService())
+                                       .addService(ProtoReflectionService.newInstance())
+                                       .build());
+    ...
+    Server server = sb.build();
+    server.start();
+
+For more information, see the official `Link tutorial <https://github.com/grpc/grpc-java/blob/master/documentation/server-reflection-tutorial.md>`_.
```

Comment:
nit: How about changing from `Link tutorial` to `gRPC Server Reflection Tutorial`?

Reply:
Thanks - fixed. Realized I don't know rst at all. Would it be possible to migrate the docs to Markdown for easier contributions?

Reply:
Happened to find https://www.sphinx-doc.org/en/master/usage/markdown.html

Reply:
Yeah, but Markdown has never been the first citizen, so you'd better learn from other .rst pages in this project and ask us at Slack. :wink: